api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -170,7 +168,6 @@ public long getQueryBatchSize() {
         (rs, unused) ->
             new ReportingInstitution()
                 .displayName(rs.getString("display_name"))
-                .duaTypeEnum(institutionDUATypeFromStorage(rs.getShort("dua_type_enum")))
```

Comment:
Tested locally - this reporting field is set to NULL, as expected.

Please also update the column description in workbench-terraform-modules to indicate that this is deprecated.

Reply:
It has been there for a while https://github.com/all-of-us/workbench-terraform-modules/blob/main/modules/workbench/modules/reporting/assets/schemas/institution.json#L16

Reply:
ah thanks!  I was looking at my outdated local copy